test/index.js

```diff
@@ -0,0 +1,40 @@
+const assert = require('assert');
+const sinon = require('sinon');
+
+const modulePath = '../src/index';
+
+/* eslint-disable global-require, import/no-dynamic-require, no-console */
+describe('entry point', () => {
+  beforeEach(() => {
+    // removing import from cache to be able to initialize the module from the very beginning
```

Comment:
Cpmment should be consistent with one here https://github.com/dial-once/node-logtify/pull/1/files#diff-569b236876bc7aa8a8d8f0647d799c5bR11